### ISSUE TITLE
Items Disappear from Spaceship After Game Relaunch

### ISSUE THREAD

**Issue by cangming99** [Feiryn/LethalMon#56]
I’ve encountered a bug where items on my ship disappear progressively after I exit and re-enter the game (this happens when I exit the game entirely, not just the save). Each time I restart, a portion of the items is missing, and eventually, all the items disappear.

Additionally, for any balls on the ship, their models remain in place, but they cannot be picked up.

After I closed LethalMon, everything returned to normal.

**Comment by Feiryn** [comment]
Can you send me your log file after loading a game with items with the mod? There may be some errors in there

**Comment by cangming99** [comment]
I saw this error log when loading the archive, maybe it's it?
```
[04:02:09.6541063] [Fatal  :DBJ.UncaughtExceptionCatcher] An uncaught exception has been caught from a mod's patcher! Details below:
[04:02:09.6814512] [Error  :DBJ.UncaughtExceptionCatcher] System.Collections.Generic.KeyNotFoundException: The given key 'BaboonHawk' was not present in the dictionary.
  at System.Collections.Generic.Dictionary`2[TKey,TValue].get_Item (TKey key) [0x0001e] in <787acc3c9a4c471ba7d971300105af24>:IL_001E
  at LethalMon.Registry+<>c__DisplayClass13_0.<GetCatchableEnemy>b__0 (System.Collections.Generic.KeyValuePair`2[TKey,TValue] cm) [0x00000] in ./Registry.cs:108
  at System.Linq.Enumerable.TryGetFirst[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] predicate, System.Boolean& found) [0x0003f] in <dab7f68612224ba3ae40f651d44f9d4c>:IL_003F
  at System.Linq.Enumerable.FirstOrDefault[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] predicate) [0x00000] in <dab7f68612224ba3ae40f651d44f9d4c>:IL_0000
  at LethalMon.Registry.GetCatchableEnemy (System.Int32 enemyTypeId) [0x0000d] in ./Registry.cs:108
  at LethalMon.Items.PokeballItem.LoadItemSaveData (System.Int32 saveData) [0x00007] in ./Items/PokeballItem.cs:284
  at (wrapper dynamic-method) StartOfRound.DMD<StartOfRound::LoadShipGrabbableItems>(StartOfRound)
  ```
  
Here is the complete log：[LogOutput.log](https://github.com/user-attachments/files/17353027/LogOutput.log)

**Comment by Feiryn** [comment]
Thanks, it will be fixed in version 0.6.6 that is coming soon
